container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -190,7 +190,7 @@ instance_groups:
             type: persistent
             tag: mysql-data
             size: 20
-          memory: 2950
+          memory: 2500
```

Comment:
`mysql-proxy`, by itself, takes 2.5GiB? And `mysql` (the actual database) has no limits?
Something sounds really wrong. (Probably in how we do limits)

The nearest round number is 2560.